app/services/places-import.js

```diff
@@ -80,7 +82,7 @@ export class PlacesImporter {
 
   static async importFromPlaces(pathToPlaces, datomStorage, limit = undefined) {
     // The WAL is enabled, so opening read-only leaves orphaned .shm/.wal files.
-    const placesDB = await sqlite.DB.open(pathToPlaces);
+    const placesDB = await sqlite.DB.open(pathToPlaces, undefined, logger);
```

Comment:
Would prefer to have an "options" object that takes in `{ mode, logger }` instead of the spread out params with undefined in there